addon/components/frost-select-dropdown.js

```diff
@@ -196,6 +215,25 @@ export default Component.extend({
       return {
         className: classNames.join(' '),
         label: get(item, 'label'),
+        secondaryLabels: get(item, 'secondaryLabels'),
```

Comment:
You already looked this property up on line 201. Instead this line can simply be:
```
secondaryLabels,
```